packages/ckeditor5-utils/src/focustracker.ts

```diff
@@ -66,16 +76,92 @@ export default class FocusTracker extends /* #__PURE__ */ DomEmitterMixin( /* #_
 	}
 
 	/**
-	 * List of registered elements.
+	 * List of registered DOM elements.
+	 *
+	 * **Note**: The list does do not include elements from {@link #externalFocusTrackers}.
 	 */
 	public get elements(): Array<Element> {
 		return Array.from( this._elements.values() );
 	}
 
 	/**
-	 * Starts tracking the specified element.
+	 * List of external focus trackers that contribute to the state of this focus tracker. See {@link #add} to learn more.
+	 */
+	public get externalFocusTrackers(): Array<FocusTracker> {
```

Comment:
I decided I don't want to extend `#elements` with elements of external focus trackers to keep the previous API contract (avoid BCs). I came up with this property that will allow integrators to dig deeper if they want, though.

Reply:
Until now `#focusedElement` always was one of `#elements.

But now `#focusedElement` may point to a DOM element outside of `#elements` if an external FT's element is focused. I didn't want to leave `#focusedElement === null` if `#isFocused === true` in that situation so the integrators may need to access `#externalFocusTrackers` to learn more.